go.mod

```diff
@@ -9,9 +9,12 @@ require (
 	github.com/alecthomas/jsonschema v0.0.0-20220216202328-9eeeec9d044b
 	github.com/benbjohnson/clock v1.3.5
 	github.com/celestiaorg/celestia-app v1.13.0
+	github.com/celestiaorg/celestia-app/v2 v2.0.0
 	github.com/celestiaorg/go-fraud v0.2.1
 	github.com/celestiaorg/go-header v0.6.2
 	github.com/celestiaorg/go-libp2p-messenger v0.2.0
+	github.com/celestiaorg/go-square v1.1.0
+	github.com/celestiaorg/go-square/merkle v0.0.0-20240117232118-fd78256df076
 	github.com/celestiaorg/nmt v0.22.0
```

Comment:
we can bump this to: https://github.com/celestiaorg/nmt/releases/tag/v0.22.1. want me to do it?

Reply:
Done in [26da487](https://github.com/celestiaorg/celestia-node/pull/3453/commits/26da487b3eb1ce547782ba8fde7c7a8a9712a9bd)